app/i18n/en/gen.php

```diff
@@ -156,6 +156,7 @@
 		'damn' => 'Damn!',
 		'default_category' => 'Uncategorized',
 		'no' => 'No',
+		'not_applicable' => 'N/A',
```

Comment:
I always found "N/A" too formal and a bit ugly. I find "Not applicable" a lot better.